nsqd/nsqd_test.go

```diff
@@ -157,6 +158,73 @@ func TestStartup(t *testing.T) {
 	<-doneExitChan
 }
 
+func TestMaxDiskSpace(t *testing.T) {
+	iterations := 2048
+	doneExitChan := make(chan int)
+
+	opts := NewOptions()
+	opts.Logger = test.NewTestLogger(t)
+	opts.MemQueueSize = 0
+	opts.MaxBytesPerFile = 10240
```

Comment:
Do we need to specify "MaxBytesPerFile"?

Reply:
When file starts to rollout?

Reply:
yes, in this test case, since by default maxBytesPerFile is much larger up to 100mb